examples/examples/custom_type_derives.rs

```diff
@@ -20,7 +20,7 @@
     runtime_metadata_path = "examples/polkadot_metadata.scale",
     // We can add (certain) custom derives to the generated types by providing
     // a comma separated list to the below attribute. Most useful for adding `Clone`:
-    generated_type_derives = "Clone, Hash"
+    generated_type_derives = "Clone"
```

Comment:
Interesting; did `Hash` not work? (I think I'd have more than one if possible just so that people can see how to add multiple derives (and adding just one is fairly obvious from that :))